docs/configuration/basic_usage.md

```diff
@@ -83,8 +83,6 @@ Sometimes it might be desirable to release each module (or just some
 modules) of multi-module project separately. If so, please make sure
 that:
 
--   tag prefixes for each module do not overlap, i.e.
```

Comment:
while your change makes axion more flexible, it does not solve all problems. This warning was created because of common scenario with projects like:

```
my-service
my-service-client
```

Because version separator and character used in names of projects is the same, it will still fail. I think you should change this warning to illustrate it instead of removing it altogether.

Reply:
I have expanded the docs by adding in an explicit warning for this issue.